### PR TITLE
Update branding and company name #3660

### PR DESCRIPTION
**Pull request by 0xE69**
Uploaded new branding images. 
Updated html_favicon_url & html_logo_url
Updated "CasperLabs" to "Casper Labs" in documentation. 

https://github.com/casper-network/casper-node/issues/3660

No Known issues. 

this PR appears to have been abandoned.   We opened an alternate PR for the changes and it is being merged - this PR is now a duplicate

### CODE REVIEW

**Comment by RitaMAllenCA** [comment]
@piotr-dziubecki Is this someone from one of your project teams? Please advise so we can get this merged, thanks

**Review by piotr-dziubecki — COMMENTED** [review]
changes look good to me

File: resources/test/rpc_schema_hashing.json [hunk 1]
```diff
@@ -3379,7 +3379,7 @@
             "description": "Representation of a value stored in global state.\n\n`Account`, `Contract` and `ContractPackage` have their own `json_compatibility` representations (see their docs for further info).",
             "anyOf": [
               {
-                "description": "A CasperLabs value.",
+                "description": "A Casper Labs value.",
```
Comment (EdHastingsCasperAssociation) [inline resources/test/rpc_schema_hashing.json]:
```suggestion
                "description": "An instance of any valid variant of StoredValue.",
```

File: node/src/types/json_compatibility/stored_value.rs [hunk 2]
```diff
@@ -25,7 +25,7 @@ use super::{Account, Contract, ContractPackage};
 #[derive(Clone, Eq, PartialEq, Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(deny_unknown_fields)]
 pub enum StoredValue {
-    /// A CasperLabs value.
+    /// A Casper Labs value.
```
Comment (EdHastingsCasperAssociation) [inline node/src/types/json_compatibility/stored_value.rs]:
```suggestion
    /// An instance of any variant of CLValue.
```

**Review by EdHastingsCasperAssociation — APPROVED** [review]

**Comment by rafal-ch** [comment]
Merging [this one](https://github.com/0xE69/casper-node/pull/1) should fix the build.

**Comment by ACStone-MTS** [comment]
@0xE69 Is there anything blocking this merge on your end, [other than this build fix](https://github.com/0xE69/casper-node/pull/1)?

File: ci/build_client_rpm.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 
-# Images used in this script are build in CasperLabs/buildenv repo
+# Images used in this script are build in Casper Labs/buildenv repo
```
Comment (MParlikar) [inline ci/build_client_rpm.sh:4]: Should just say 'Casper' not Casper Labs

File: ci/casper_updater/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 authors = ["Fraser Hutchison <fraser@casperlabs.io>"]
-description = "A tool to update versions of all published CasperLabs packages."
+description = "A tool to update versions of all published Casper Labs packages."
```
Comment (MParlikar) [inline ci/casper_updater/Cargo.toml:3]: Should say just 'Casper' - Not Casper Labs actually.

File: ci/casper_updater/README.md [hunk 5]
```diff
@@ -1,10 +1,10 @@
 # casper-updater
 
-A tool to update versions of all published CasperLabs packages.
+A tool to update versions of all published Casper Labs packages.
```
Comment (MParlikar) [inline ci/casper_updater/README.md:3]: Same here - Casper

File: ci/casper_updater/README.md [hunk 6]
```diff
@@ -1,10 +1,10 @@
...
 
 # Usage
 
-The tool iterates through each published CasperLabs package, asking for a new version for each or automatically bumping the major, minor or patch version if `--bump=[major|minor|patch]` was specified.  Once a valid version is specified, all files dependent on that version are updated.
+The tool iterates through each published Casper Labs package, asking for a new version for each or automatically bumping the major, minor or patch version if `--bump=[major|minor|patch]` was specified.  Once a valid version is specified, all files dependent on that version are updated.
```
Comment (MParlikar) [inline ci/casper_updater/README.md:7]: Just remove Casper Labs all together here.

File: ci/casper_updater/src/dependent_file.rs [hunk 7]
```diff
@@ -5,7 +5,7 @@ use std::{
 
 use regex::Regex;
 
-/// A file which is dependent on the version of a certain CasperLabs crate.
+/// A file which is dependent on the version of a certain Casper Labs crate.
```
Comment (MParlikar) [inline ci/casper_updater/src/dependent_file.rs:8]: Rename to Casper

File: ci/casper_updater/src/main.rs [hunk 8]
```diff
@@ -1,4 +1,4 @@
-//! A tool to update versions of all published CasperLabs packages.
+//! A tool to update versions of all published Casper Labs packages.
```
Comment (MParlikar) [inline ci/casper_updater/src/main.rs:1]: rename to Casper

File: ci/casper_updater/src/package.rs [hunk 9]
```diff
@@ -18,7 +18,7 @@ use crate::{
 
 const CAPTURE_INDEX: usize = 2;
 
-/// Represents a published CasperLabs crate or AssemblyScript package which may need its version
+/// Represents a published Casper Labs crate or AssemblyScript package which may need its version
```
Comment (MParlikar) [inline ci/casper_updater/src/package.rs:21]: re label to 'Casper'

File: docker_make.sh [hunk 10]
```diff
@@ -1,5 +1,5 @@
 
-# Images used in this script are build in CasperLabs/buildenv repo
+# Images used in this script are build in Casper Labs/buildenv repo
```
Comment (MParlikar) [inline docker_make.sh:2]: Rename to 'Casper'

File: execution_engine/Cargo.toml [hunk 11]
```diff
@@ -3,7 +3,7 @@ name = "casper-execution-engine"
 version = "3.0.0" # when updating, also update 'html_root_url' in lib.rs
 authors = ["Henry Till <henrytill@gmail.com>", "Ed Hastings <ed@casperlabs.io>"]
 edition = "2018"
-description = "CasperLabs execution engine crates."
+description = "Casper Labs execution engine crates."
```
Comment (MParlikar) [inline execution_engine/Cargo.toml:6]: Rename to 'Casper'

File: execution_engine/README.md [hunk 12]
```diff
@@ -7,7 +7,7 @@
 [![Documentation](https://docs.rs/casper-execution-engine/badge.svg)](https://docs.rs/casper-execution-engine)
 [![License](https://img.shields.io/badge/license-Apache-blue)](https://github.com/casper-network/casper-node/blob/master/LICENSE)
 
-The main component of the CasperLabs Wasm execution engine.
+The main component of the Casper Labs Wasm execution engine.
```
Comment (MParlikar) [inline execution_engine/README.md:10]: Rename to 'Casper'

File: execution_engine/src/lib.rs [hunk 13]
```diff
@@ -2,8 +2,8 @@
 
 #![doc(html_root_url = "https://docs.rs/casper-execution-engine/3.0.0")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```
Comment (MParlikar) [inline execution_engine/src/lib.rs:6]: Can we put the Casper logo here?  The source code is owned by Casper Assocation, not Labs.

File: execution_engine_testing/test_support/src/lib.rs [hunk 14]
```diff
@@ -2,8 +2,8 @@
 
 #![doc(html_root_url = "https://docs.rs/casper-engine-test-support/3.0.0")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```
Comment (MParlikar) [inline execution_engine_testing/test_support/src/lib.rs:6]: Can we put the Casper logo here?  The source code is owned by Casper Assocation, not Labs.

File: node/src/components/consensus/cl_context.rs [hunk 15]
```diff
@@ -54,7 +54,7 @@ impl ConsensusValueT for Arc<BlockPayload> {
     }
 }
 
-/// The collection of types used for cryptography, IDs and blocks in the CasperLabs node.
+/// The collection of types used for cryptography, IDs and blocks in the Casper Labs node.
```
Comment (MParlikar) [inline node/src/components/consensus/cl_context.rs:57]: Casper

File: node/src/components/rpc_server/rpcs/docs.rs [hunk 16]
```diff
@@ -39,7 +39,7 @@ pub(crate) static OPEN_RPC_SCHEMA: Lazy<OpenRpcSchema> = Lazy::new(|| {
         url: "https://casperlabs.io".to_string(),
     };
     let license = OpenRpcLicenseField {
-        name: "CasperLabs Open Source License Version 1.0".to_string(),
+        name: "Casper Labs Open Source License Version 1.0".to_string(),
```
Comment (MParlikar) [inline node/src/components/rpc_server/rpcs/docs.rs:42]: Casper

File: node/src/components/rpc_server/rpcs/docs.rs [hunk 17]
```diff
@@ -498,7 +498,7 @@ mod tests {
             url: "https://casperlabs.io".to_string(),
         };
         let license = OpenRpcLicenseField {
-            name: "CasperLabs Open Source License Version 1.0".to_string(),
+            name: "Casper Labs Open Source License Version 1.0".to_string(),
```
Comment (MParlikar) [inline node/src/components/rpc_server/rpcs/docs.rs:501]: The license should be Apache 2.0 - not COSL

File: node/src/lib.rs [hunk 18]
```diff
@@ -10,8 +10,8 @@
 
 #![doc(html_root_url = "https://docs.rs/casper-node/1.4.13")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```
Comment (MParlikar) [inline node/src/lib.rs:14]: Casper Association

File: resources/test/rpc_schema_hashing.json [hunk 19]
```diff
@@ -9,11 +9,11 @@
         "title": "Client API of Casper Node",
         "description": "This describes the JSON-RPC 2.0 API of a node on the Casper network.",
         "contact": {
-          "name": "CasperLabs",
+          "name": "Casper Labs",
           "url": "https://casperlabs.io"
         },
         "license": {
-          "name": "CasperLabs Open Source License Version 1.0",
+          "name": "Casper Labs Open Source License Version 1.0",
```
Comment (MParlikar) [inline resources/test/rpc_schema_hashing.json:16]: This should be Apache 2.0

File: smart_contracts/contract/src/lib.rs [hunk 20]
```diff
@@ -52,8 +52,8 @@
 )]
 #![doc(html_root_url = "https://docs.rs/casper-contract/1.4.4")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```
Comment (MParlikar) [inline smart_contracts/contract/src/lib.rs:56]: Casper Association

File: types/src/lib.rs [hunk 21]
```diff
@@ -12,8 +12,8 @@
 )]
 #![doc(html_root_url = "https://docs.rs/casper-types/1.5.0")]
 #![doc(
-    html_favicon_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Favicon_RGB_50px.png",
-    html_logo_url = "https://raw.githubusercontent.com/CasperLabs/casper-node/master/images/CasperLabs_Logo_Symbol_RGB.png",
+    html_favicon_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon_48.png",
+    html_logo_url = "https://raw.githubusercontent.com/casper-network/casper-node/blob/dev/images/CasperLabs_Logo_Favicon.png",
```
Comment (MParlikar) [inline types/src/lib.rs:16]: Casper Association

**Review by MParlikar — CHANGES_REQUESTED** [review]
The Association owns the software, instances referring to Labs, for the most part should be updated to refer to the Association, references to the tech, should state just 'Casper' where possible.  License is Apache 2.0

**Pull request closed without merging**